### ISSUE TITLE
Optional Chaining Wrongly Formatted.

### ISSUE THREAD

**Issue by bhavya32** [sunjw/jstoolnpp#108]
Sample Code - 
```
const adventurer = {
 dog: {
    name: 'cat1'
  }
};

const dogName = adventurer.dog?.name;
console.log(dogName);
```
If I apply JSFormat in NPP ```which is an excellent plugin btw :)```, I formats it the wrong way.

Adds unwanted spaces in - ```const dogName = adventurer.dog ? .name;``` Causing error as there should me no spaces in optional chaining. @sunjw Please see if you can fix this whenever you are free.

**Comment by sunjw** [comment]
Thanks  for reporting this. I'll find time to fix.

**Comment by sunjw** [comment]
Fixed after 1e6e69a2b844dd0d8b4982d592a30ee26fe180de.

**Comment by bhavya32** [comment]
Thank you. Appreciate your time.
